patches/amd-mainline/HIP/0001-Fix-FindHIP.cmake-search-for-hipconfig.patch

```diff
@@ -0,0 +1,49 @@
+From 027cb4a11e2b6c47606308b01e8180507496701d Mon Sep 17 00:00:00 2001
+From: Aaryaman Vasishta <jem456.vasishta@gmail.com>
+Date: Tue, 6 May 2025 15:07:26 +0530
+Subject: [PATCH] Fix FindHIP.cmake search for hipconfig.
+
+---
+ cmake/FindHIP.cmake | 11 ++---------
+ 1 file changed, 2 insertions(+), 9 deletions(-)
+
+diff --git a/cmake/FindHIP.cmake b/cmake/FindHIP.cmake
+index 422b7089..a93e839f 100644
+--- a/cmake/FindHIP.cmake
++++ b/cmake/FindHIP.cmake
+@@ -99,7 +99,7 @@ if(NOT APPLE)
+         # Search in user specified path first
+         find_path(
+             HIP_ROOT_DIR
+-            NAMES bin/hipconfig
++            NAMES bin/hipconfig${CMAKE_EXECUTABLE_SUFFIX}
```

Comment:
From the code below it seem one was looking for a `.bat` but following the docs [CMAKE_EXECUTABLE_SUFFIX](https://cmake.org/cmake/help/latest/variable/CMAKE_EXECUTABLE_SUFFIX.html), refers to `.exe` on Windows. Is this the expected behavior then?

Reply:
Ah I should have read the PR description more carefully, as it is!

Reply:
I also added more context on the upstream PR I sent.